approvals/src/main/java/no/sikt/nva/approvals/persistence/DynamoDbConstants.java

```diff
@@ -0,0 +1,16 @@
+package no.sikt.nva.approvals.persistence;
+
+public final class DynamoDbConstants {
+
+    public static final String TABLE_NAME = "TABLE";
```

Comment:
I think we need to read this value from an environment variable in production code. Doesn't the name have to match the resource name in the template?

Great if you can keep it static or as an input parameter for `DynamoDbLocal` in the tests though, because reading the table name from an environment variable makes test setup difficult.